code/datums/status_effects/debuffs.dm

```diff
@@ -76,8 +76,8 @@
 				healing -= 0.1
 		owner.adjustBruteLoss(healing)
 		owner.adjustFireLoss(healing)
-		owner.adjustToxLoss(healing)
-		owner.adjustStaminaLoss(healing, 1, 1)
+		owner.adjustToxLoss(healing, 1, 1)
```

Comment:
i know im guilty of oofs too but these should be booleans my dude

Reply:
done